site/generate-jenkins-io-template.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+WWW_ROOT_DIR="${WWW_ROOT_DIR:-${1:-./www2}}"
+SELF_URL="${SELF_URL:-${2:-https://updates.jenkins.io}}"
+
+wget -q --convert-links -O "$WWW_ROOT_DIR/index.html" --convert-links https://www.jenkins.io/templates/updates/index.html
+sed -i -e "s|<jio-navbar.*/jio-navbar>|<jio-navbar class='fixed-top' id='ji-toolbar' property='${SELF_URL}'></jio-navbar>|g" "$WWW_ROOT_DIR/index.html"
+sed -i -e "s|<jio-footer.*/jio-footer>|<jio-footer property='${SELF_URL}'></jio-footer>|g" "$WWW_ROOT_DIR/index.html"
```

Comment:
Why is this a separate `sed` call? Can't this be an additional expression in the first `sed` call? Readability perhaps?

Reply:
yea i guess, it never really occured to me to join them, i found it easier to read i guess?